tripal/includes/tripal.user.inc

```diff
@@ -3,47 +3,47 @@
 
 function tripal_user_get_files($uid){
   $directory = [];
-  
+
   // Get all of the files that have been uploaded by the user.
   $sql = "
      SELECT FM.fid, FM.filename, FM.uri, FM.uid, FM.filesize, TGEF.expiration_date
      FROM {file_managed} FM
        INNER JOIN {file_usage} FU on FM.fid = FU.fid and FM.uid = :user_id
        LEFT JOIN {tripal_expiration_files} TGEF on TGEF.fid = FU.fid
      WHERE FU.module = 'tripal'
-     ORDER BY FM.filename
+     ORDER BY FM.uri
```

Comment:
Whoops. I didn't mean for this fix to make it in to this PR.  But it's so simple I'll leave it.  It just fixes a sorting issue with the user's uploaded file list.